tests/parametric/test_otel_env_vars.py

```diff
@@ -0,0 +1,234 @@
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "golang", reason="Not implemented")
+    @missing_feature(context.library < "nodejs@5.11.0", reason="Implemented in v5.11.0, v4.35.0, &v3.56.0")
+    @missing_feature(context.library == "ruby", reason="Not implemented")
+    @missing_feature(context.library == "php", reason="Not implemented")
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_SAMPLER": "parentbased_always_off",}],
+    )
+    def test_otel_traces_parentbased_off(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        if context.library != "nodejs":
+            assert resp["dd_trace_sample_ignore_parent"] == False
+        assert resp["dd_trace_sample_rate"] == 0.0
+
+    @missing_feature(context.library == "dotnet", reason="Not implemented")
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "golang", reason="Not implemented")
+    @missing_feature(context.library < "nodejs@5.11.0", reason="Implemented in v5.11.0, v4.35.0, &v3.56.0")
+    @missing_feature(context.library == "ruby", reason="Not implemented")
+    @missing_feature(context.library == "php", reason="Not implemented")
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_SAMPLER": "parentbased_traceidratio", "OTEL_TRACES_SAMPLER_ARG": "0.1"}],
+    )
+    def test_otel_traces_parentbased_ratio(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        if context.library != "nodejs":
+            assert resp["dd_trace_sample_ignore_parent"] == False
+        assert resp["dd_trace_sample_rate"] == 0.1
+
+    @missing_feature(context.library == "dotnet", reason="Not implemented")
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "golang", reason="Not implemented")
+    @missing_feature(context.library < "nodejs@5.11.0", reason="Implemented in v5.11.0, v4.35.0, &v3.56.0")
+    @missing_feature(context.library == "ruby", reason="Not implemented")
+    @missing_feature(context.library == "php", reason="Not implemented")
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_EXPORTER": "none"}],
+    )
+    def test_otel_traces_exporter_none(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert resp["dd_trace_enabled"] == False
+
+    @missing_feature(context.library == "dotnet", reason="Not implemented")
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "golang", reason="Not implemented")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @missing_feature(context.library == "ruby", reason="Not implemented")
+    @missing_feature(context.library == "php", reason="Not implemented")
```

Comment:
If we are setting "missing_feature" in the manifests, and we are going to skip all the class methods, Do we need all these decorators?